Server/Components/Recordings/recordings_main.cpp

```diff
@@ -148,24 +124,7 @@ class RecordingsComponent final : public IRecordingsComponent, public PlayerConn
 			{
 				uint32_t timeSinceRecordStart = duration_cast<Milliseconds>(Time::now() - data->start_).count();
```

Comment:
Nitpicking, but if you make it `const` for the on foot code, it can become `const` here as well.